raco/backends/myria/myria.py

```diff
@@ -2437,6 +2448,11 @@ def compile_to_json(raw_query, logical_plan, physical_plan,
     assert isinstance(physical_plan, subplan_ops), \
         'Physical plan must be a subplan operator, not {}'.format(type(physical_plan))  # noqa
 
+    if('OrderBy' in str(logical_plan)
```

Comment:
**1)** I implemented this approach (checking whether the Root has an Order By child), but I'm not sure it would always work. For example, for this query:

```
T1 = [from scan(TwitterK) as t ORDERBY $0 ASC, $1 DESC emit $0];  store(T1, TwitterKOrdered);
```

we would have the following logical plan (where the Apply operator ends up being the child of the store): 
`Store(public:adhoc:TwitterKOrdered)[Apply(a=$0)[OrderBy(0+,1-)[Scan(public:adhoc:TwitterK)]]]`

**2)** There's also this other case I'm testing with:
```
T1 = [from scan(TwitterK) as t ORDERBY $0 ASC, $1 DESC emit $0, count(*)];
T2 = LIMIT(T1, 10);
store(T2, TwitterKOrdered);
```
I don't remember if we wanted to support OrderBy with aggregates, but I don't think our rules would apply here. We would need a plan like this:
`store -> limit(global) -> aggregate -> orderby (global) -> collect ->scan`

I don't believe it would be correct to apply the limit and orderby locally for this case, right?